CMakeLists.txt

```diff
@@ -100,7 +103,16 @@ if ("${NSYNC_LANGUAGE}X" STREQUAL "c++11X")
 		)
 		set (NSYNC_TEST_OS_SRC
 			"platform/posix/src/start_thread.c"
+        )
+    elseif ( "${CMAKE_SYSTEM_NAME}X" STREQUAL "AndroidX")
+		include_directories ("${PROJECT_SOURCE_DIR}/platform/posix")
+		add_compile_options ("-std=c++11")
```

Comment:
Belongs to toolchain, no need to set it here.

Reply:
I agree in theory.  In this case I was aiming for a minimal patch (replicating existing syntax/logic).    Do you prefer updating the CMakeLists.txt?

Reply:
> Do you prefer updating the rest of the CMakeLists.txt?

No, minimal patch seems fine. Just no need to add new line with `add_compile_options ("-std=c++11")`.

Reply:
Ok.  For now I'll leave it as is?  Let me know if you still think a change is required in this PR.

Reply:
> For now I'll leave it as is?

Ok